.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
This one bro.

Reply:
Alright, i just removed it and push it, plz check